tests/test_appipe.py

```diff
@@ -43,8 +43,8 @@ class PipelineTestSuite(lsst.utils.tests.TestCase):
     def _makeDefaultConfig(cls):
         config = ApPipeTask.ConfigClass()
         config.load(os.path.join(cls.datadir, "config", "apPipe.py"))
```

Comment:
I'm pretty sure the config file does not need to be updated, but did you run unit tests with `ap_pipe_testdata` installed?

Reply:
I did. Both on lsst-dev and on Jenkins. Seemed to have ran okay though I'm not going to say that I looked terribly deeply into the unittests to make sure they weren't missing something.

Reply:
Have reconfirmed on lsst-dev that both `ap_pipe` and `ap_verify` pass all tests with their respective testdata setup.